js/renderingPicture.js

```diff
@@ -0,0 +1,37 @@
+import {createPhotos} from './mockPhotos.js';
+
+const dataCreatePicture = createPhotos();
+const SAMPLE = document.querySelector('#picture').content.cloneNode(true);
+const PICTURES = document.querySelector('.pictures');
+const fragment = document.createDocumentFragment();
+
+const createPictures = (data, sample) =>{
+  const sampleClone = sample.cloneNode(true);
+  const randomPhoto = sampleClone.querySelector('.picture__img');
+  randomPhoto.srс = data.url;
+  randomPhoto.alt = data.description;
+  const likes = sampleClone.querySelector('.picture__likes');
+  likes.textContent = data.likes;
+  const comments = sampleClone.querySelector('.picture__comments');
+  comments.textContent = data.comments.length;
+  fragment.appendChild(sampleClone);
```

Comment:
Итак, в этой функции мы сколнировали элемент, заполнили его данными, но давай не добавлять этот элемент в фрагмент, а вернем наш элемент из этой функции, сделав её попроще (в логическом плане):
```suggestion
  return sampleClone;
```
Это даст нам больше гибкости в дальнейшем: когда функция (любая) выполняет одну и только одну задачу -- это идеальный случай, к сожалению, не всегда это удается сделать, но тут есть такая возможность: наша функция `createPicture` получает данные и возвращает нам DOM элемент заполненный этими данными. Теперь мы можем её использовать где угодно и как угодно для этой задачи в рамках нашего проекта.